carbonmark-api/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@klimadao/carbonmark-api",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Can you bump this to 
```suggestion
  "version": "2.1.0",
```

Just because we are adding functionality with the new route.

From semver.org

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```